<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConductivityEditor</class>
 <widget class="QWidget" name="ConductivityEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="startingValueLabel">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The value that will be assigned to the selected nodes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Starting Value</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="startingValueDoubleSpinBox"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="incrementAmountLabel">
     <property name="text">
      <string>Increment Amount</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="incrementAmountDoubleSpinBox"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="distanceLabel">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The starting value will be incremented by Increment Amount for each distance.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Distance</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="distanceSpinBox">
     <property name="maximum">
      <number>999</number>
     </property>
     <property name="value">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="valueMaximumLabel">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;After this value has been reached all cells farther away will have this conductivity&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Value Maximum</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="valueMaximumDoubleSpinBox">
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>99.989999999999995</double>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="applyButton">
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="percValuesCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Gap Junction in %</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
